_data/glossary/statefulset.yml

```diff
@@ -1,18 +1,18 @@
 id: statefulset
 name: StatefulSet
-formerly:
+aka:
 - PetSet
 related:
-- Deployment
-- Pod
+- deployment
+- pod
 tags:
-- Storage
-- Workload
-- API Object
+- core-object
+- workload
+- storage
 short-description: |
-  Manage the deployment and scaling of a set of Pods, *and provide guarantees about ordering*. They do so by maintaining a *unique*, sticky identity for each of their Pods.
+  Manages the deployment and scaling of a set of Pods, *and provides guarantees about the ordering and uniqueness* of these Pods.
 
 long-description: |
-  Like Deployments, StatefulSets manage Pods that are based on an identical container spec. However, although their specs are the same, the Pods in a StatefulSet are not interchangeable. Each Pod has a persistent identifier that it maintains across any rescheduling.
+  Like a Deployment, a StatefulSet manages Pods that are based on an identical container spec. Unlike a Deployment, a StatefulSet maintains a sticky identity for each of their Pods. These pods are created from the same spec, but are not interchangeable--each has a persistent identifier that it maintains across any rescheduling.
 
-  StatefulSets also operate according to the Controller pattern. You define your desired state in a StatefulSet *object*, and the StatefulSet *controller* makes any necessary updates to the get there from the current state.
+  A StatefulSet operates under the same pattern as any other Controller. You define your desired state in a StatefulSet *object*, and the StatefulSet *controller* makes any necessary updates to get there from the current state.
```

Comment:
Consistent caps: Controller/controller

Reply:
I feel like this capitalization denotes Controller as something special in K8s world, as opposed to some sort of other meaning of "controller" (e.g. MVC controller)?

Reply:
That's real. I think "Controller" works here because it distinguishes not strictly from other controllers (e.g. MVC) in general, but among other K8s Controllers as well. (e.g., Admission, Replication)